colorGame.js

```diff
@@ -8,6 +8,10 @@ var colors = [
 ];
 
 var $squares = document.querySelectorAll(".square");
+var pickedColor = colors[3];
```

Comment:
3 is a magical number, you should put it in a constant that describes what does it represent.